doc/chapters/api/index.rst

```diff
@@ -3,38 +3,41 @@
 API
 ***
 
-This chapter documents the D-Bus API for interacting with the SoftwareContainerAgent. This is
-mostly useful for integrators.
+This chapter documents the D-Bus API for interacting with the
+SoftwareContainerAgent. This is mostly useful for integrators.
 
 .. _dbus-api:
 
 D-Bus API
 =========
 
-D-Bus API is an IPC interface to call SoftwareContainer agent methods. The API provides following object path and interface.
+D-Bus API is an IPC interface to call SoftwareContainer agent methods. The API
+provides following object path and interface.
 
 :Object Path: /com/pelagicore/SoftwareContainer
 :Interface: com.pelagicore.SoftwareContainerAgent
 
 Methods
 -------
-All methods that modify the state of the containers return a ``success`` variable along with the
-actual return value. This is to not have to rely on magic values. The two methods ``List`` and
-``ListCapabilities`` are the only ones that doesn't modify the state.
+All methods that modify the state of the containers return a ``success``
+variable along with the actual return value. This is to not have to rely on
+magic values. The two methods ``List`` and ``ListCapabilities`` are the only
+ones that doesn't modify the state.
 
-List
-----
-Returns a list of the current containers
+BindMount
+---------
+Binds a directory on the host to the container.
 
-:Return Values:
-        :containers: ``array<int32>`` IDs for all containers
+:Parameters:
+        :containerID: ``int32`` The ID obtained by CreateContainer method.
+        :pathInHost: ``string`` absolute path to the directory in the host.
+        :pathInContainer: ``string`` the absolute path to the directory in container.
+        :readOnly: ``bool`` indicates whether the directory is read-only or not.
 
-ListCapabilities
-----------------
-Lists all capabilities that the user can apply.
+|
 
-:Return Value:
-        :capabilities: ``array<string>`` all available capability names
+:Return Values:
```

Comment:
I expected that that was the case. Thought it would be good to change it at this time. :)